examples/CRISP/scripts/cli.sh

```
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -e
+
+./scripts/tasks/dockerize.sh ./scripts/tasks/cli.sh $@
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)